deploy/charts/litellm-helm/index.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+entries:
+  postgresql:
```

Comment:
hey @lowjiansheng what's the point of this file?

Reply:
It contains the metadata of all the charts and their deps in the repo. Not sure if it is necessary, but it is generated when run `helm repo index`